QRCodeReader.swift.podspec

```diff
@@ -10,9 +10,10 @@ Pod::Spec.new do |s|
   s.source           = { :git => 'https://github.com/yannickl/QRCodeReader.swift.git', :tag => s.version }
   s.screenshot       = 'http://yannickloriot.com/resources/qrcodereader.swift-screenshot.jpg'
 
-  s.ios.deployment_target = '8.0'
+  s.ios.deployment_target = '10.0'
```

Comment:
Is there a reason to make this `10.0`? Eventually we want to support `9.0`, so if possible it would be better to make this `8.0` or `9.0` instead of `10.0`.

Reply:
Honestly, don't remember why I set it, reverted back to 8.0